app/components/SignInForm/SignInForm.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import BorderWithIcon from '../BorderWithIcon/BorderWithIcon';
+
+const styles = theme => ({
+  main: {
+    width: 'auto',
+    display: 'block', // Fix IE 11 issue.
+    marginLeft: theme.spacing.unit * 3,
+    marginRight: theme.spacing.unit * 3,
+    [theme.breakpoints.up(400 + theme.spacing.unit * 3 * 2)]: {
+      width: 400,
+      marginLeft: 'auto',
+      marginRight: 'auto',
+    },
+    form: {
+      width: '100%', // Fix IE 11 issue.
+      marginTop: theme.spacing.unit,
+    },
+    submit: {
+      marginTop: theme.spacing.unit * 3,
+    },
+  },
+});
+
+const SignInForm = (props) => {
+  const { classes, onEmailChange, onPasswordChange, onCheckboxChange, onButtonClick } = props;
+  return (
+    <main className={classes.main}>
+      <CssBaseline />
+      <BorderWithIcon text="Sign In">
+        <LockOutlinedIcon />
+        <form className={classes.form}>
```

Comment:
This component should start at this line. The component wrapping the form don't have anything to do with the form. The idea is that you should be able to use this form in another context, which is not possible now.